File: kata/8kyu/the-wide-mouthed-frog/the-wide-mouthed-frog.js
// https://www.codewars.com/kata/57ec8bd8f670e9a47a000f89

const mouthSize = (animal) => {
  return animal.toLowerCase() === 'alligator' ? 'small' : 'wide'
}

mouthSize('toucan') // -> "wide"
mouthSize('ant bear') // -> "wide"
mouthSize('alligator') // -> "small"

File: kata/8kyu/the-wide-mouthed-frog/the-wide-mouthed-frog.py
# https://www.codewars.com/kata/57ec8bd8f670e9a47a000f89

def mouth_size(animal): 
    return "small" if animal.lower() == "alligator" else "wide"

mouth_size("toucan") # -> "wide"
mouth_size("ant bear") # -> "wide"
mouth_size("alligator") # -> "small"